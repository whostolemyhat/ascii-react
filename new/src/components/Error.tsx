export const Error = ({ message }: { message: string }) => (
  <div className="error-message">{message}</div>
);
