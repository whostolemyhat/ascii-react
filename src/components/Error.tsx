export const Error = ({ message }: { message: string }) => (
  <div className="error-message" data-testid="error">
    {message}
  </div>
);
