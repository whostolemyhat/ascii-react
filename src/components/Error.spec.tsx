import { render, screen } from '@testing-library/react';
import { Error } from './Error';

describe('Error component', () => {
  it('should render message', () => {
    render(<Error message="uh oh" />);
    expect(screen.getByTestId('error').textContent).toEqual('uh oh');
    expect(screen.getByTestId('error')).toMatchSnapshot();
  });
});
