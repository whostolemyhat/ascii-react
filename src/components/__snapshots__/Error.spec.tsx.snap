// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error component should render message 1`] = `
<div
  class="error-message"
  data-testid="error"
>
  uh oh
</div>
`;
